src/protocolgame.cpp

```diff
@@ -120,7 +119,7 @@ void ProtocolGame::login(const std::string& name, uint32_t accountId, OperatingS
 			auto output = OutputMessagePool::getOutputMessage();
 			output->addByte(0x16);
 			output->addString(ss.str());
-			output->addByte(retryTime);
+			output->addByte(static_cast<uint8_t>(retryTime));
```

Comment:
Why is this cast needed? Also, why doesn't `getTime` return `uint8_t` straight away?

Reply:
The client probably expects 1 byte, not 8. (int64). 
But yeah, why not just return it correctly in the first placed. Fixed in my latest commit.